test/python/pulse/test_transforms.py

```diff
@@ -401,12 +407,286 @@ def test_multiple_schedules(self):
             schedule += Play(SamplePulse([0.0, 0.2]), drive_channel)
             schedules.append(schedule)
 
-        compressed_schedule = compress_pulses(schedules)
+        compressed_schedule = transforms.compress_pulses(schedules)
         original_pulse_ids = get_pulse_ids(schedules)
         compressed_pulse_ids = get_pulse_ids(compressed_schedule)
         self.assertEqual(len(original_pulse_ids), 6)
         self.assertEqual(len(compressed_pulse_ids), 2)
 
 
+class TestAlignSequential(QiskitTestCase):
+    """Test sequential alignment transform."""
+
+    def test_align_sequential(self):
+        """Test sequential alignment without a barrier."""
+        d0 = pulse.DriveChannel(0)
+        d1 = pulse.DriveChannel(1)
+
+        schedule = pulse.Schedule()
+        schedule = schedule.insert(1, instructions.Delay(3, d0), mutate=True)
+        schedule = schedule.insert(4, instructions.Delay(5, d1), mutate=True)
+        schedule = schedule.insert(12, instructions.Delay(7, d0), mutate=True)
+
+        schedule = transforms.remove_directives(transforms.align_sequential(schedule))
```

Comment:
why do you have to remove directives?

Reply:
Its used in the assembler to make sure we don't try and package directives into a Qobj (which would fail).

Reply:
Fixed in 82300005a.